dsp/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 [dependencies]
 libm = "0.2.1"
 serde = { version = "1.0", features = ["derive"], default-features = false }
+serde-json-core = "0.1"
```

Comment:
I don't think we should need this. Might be a bug in miniconf that's requiring this